src/session/command-history.js

```diff
@@ -1,5 +1,21 @@
+"use strict"
```

Comment:
This is not necessary, it is automatically added in modern environemnt

Reply:
This line was catching errors in the unit tests at least, which led me to believe this isn't being automatically added currently. Is there a way to confirm it is being added in the invocations?